DependencyInjection/CmfSeoExtension.php

```diff
@@ -27,6 +26,9 @@
  */
 class CmfSeoExtension extends Extension
 {
+    const CLASS_SEO_METADATA_PHPCR = 'Symfony\Cmf\Bundle\SeoBundle\Doctrine\Phpcr\SeoMetadata';
+    const CLASS_SEO_METADATA_ORM = 'Symfony\Cmf\Bundle\SeoBundle\Document\SeoMetadata';
```

Comment:
And i Know the reason why:
We are able to load and do unit tests based on the persistence setting,
but we have no chance with the current version of the TestinBundle to
switch configuration for FrontendTest. We have on configuration loaded
and thats the PHPCR one.